Patches/Moyo from the depth/ThingDefs_Misc/Moyo_Dragonian_Apparel.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Moyo-From the depth</li></mods>
+		<match Class="PatchOperationFindMod">
+		<mods>
+		<li>Dragonian GaramRace</li>
+		<li>Dragonian Race</li>
+		</mods>
+			<match Class="PatchOperationSequence">
+			<operations>
+				
+				<!--Dragon Dress-->
+				
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[@Name="Moyo_DragonDress"]/statBases</xpath>
+					<value>
+						<Bulk>7.5</Bulk>
+						<WornBulk>1</WornBulk>
+					</value>
+				</li>
+				
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[@Name="Moyo_DragonDress"]/statBases/ArmorRating_Sharp</xpath>
+					<value>
+						<ArmorRating_Sharp>7.5</ArmorRating_Sharp>

```

Comment:
Same remark as the other dress.

Reply:
Once again, advanced stuff that has both hyperweave and another strong modded fabric involved. Has higher armor in vanilla than the other dress, so it retains that aspect.